pkg/controller/perconaservermongodbrestore/perconaservermongodbrestore_controller.go

```diff
@@ -230,8 +230,8 @@ func (r *ReconcilePerconaServerMongoDBRestore) reconcileRestore(cr *psmdbv1.Perc
 	return nil
 }
 
-func runRestore(backup string, storage psmdbv1.BackupStorageSpec, pbmc *backup.PBM) (string, error) {
-	err := pbmc.SetConfig(storage)
+func runRestore(backup string, storage psmdbv1.BackupStorageSpec, pbmc *backup.PBM, pitr psmdbv1.PITRSpec) (string, error) {
+	err := pbmc.SetConfig(storage, pitr)
```

Comment:
according to https://www.percona.com/doc/percona-backup-mongodb/running.html#restoring-a-backup

> If you enabled Point-in-Time Recovery, disable it before running pbm restore. This is because Point-in-Time Recovery incremental backups and restore are incompatible operations and cannot be run together.

Reply:
Yes, a user has to disable it manually before a restore.

@spron-in do i get u right?

Reply:
As discussed in slack, please disable pitr before restore and enable after.